lib/graphql/connections/primary_key/asc.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module GraphQL
+  module Connections
+    # Implements pagination by one field with asc order
+    module PrimaryKey
+      class Asc < Base
+        private
+
+        def page_comparable_method(query_type:, page_type:)
+          if page_type == :previous
+            return query_type == :query ? :lt : :lteq
+          end
+
+          query_type == :query ? :gt : :gteq
+        end
+
+        def first_limited_sorted_table
+          arel_table[primary_key].asc
+        end
+
+        def last_limited_sorted_table
+          arel_table[primary_key].desc
+        end
+
+        def sliced_comparable_method(type)
+          type == :after ? :gt : :lt
```

Comment:
The same as in my previous comment - it's better to use `case when else raise`

Reply:
Here I used solution witn const, thank you for review